jobs/e2e_node/crio/crio_dra.ign

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
How is this file different from others? I suppose because it enables CDI support, but that's not obvious. A comment at the line which does that might be useful.

Reply:
CDI support is enabled by default in CRI-O.
Using separate image config and .ign file looked reasonable for me until you started to ask questions about it.
Switched to existing configuration for serial tests, thanks.